packages/common/src/FormattingUtils.ts

```diff
@@ -71,7 +71,6 @@ export const formatWithMaxDecimals = (
 };
 
 export const createFormatFunction = (
-  _web3: Web3 | undefined, // unused
```

Comment:
this field is unused. I removed it from here an all places that use this util to simplify things a bit.